deepchem/feat/base_classes.py

```diff
@@ -73,27 +117,56 @@ def _featurize_complex(self, mol_pdb, complex_pdb):
     raise NotImplementedError('Featurizer is not defined.')
 
 
-class Featurizer(object):
-  """
-  Abstract class for calculating a set of features for a molecule.
+class MolecularFeaturizer(object):
```

Comment:
This is great! I will add a `CrystalFeaturizer` superclass in my next PR.

Reply:
Why `MolecularFeaturizer` is not superclass of `Featurizer`?

Reply:
Good catch! Fixed :)

Reply:
It still lists the superclass as `object` instead of `Featurizer`.

Reply:
Oh sorry for the confusion. I was doing this on a local branch which I haven't pushed up yet. Will push up now